addons/website/static/src/js/editor/odoo_editor.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import { OdooEditor } from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
+import { patch } from "@web/core/utils/patch";
+import { removeTextHighlight } from "@website/js/text_processing";
+
+/**
+ * The goal of this patch is to correctly handle OdooEditor's behaviour for text
+ * highlight elements.
+ */
+patch(OdooEditor.prototype, {
+    /**
+     * @override
+     */
+    _onClipboardCopy(e) {
+        super._onClipboardCopy(e);
```

Comment:
I tried to copy-paste the text highlighted here, it doesnt work

![image](https://github.com/odoo/odoo/assets/52911687/a98fb0a9-3d2a-4327-b55c-f5706234d5ee)

Reply:
When I change the Size option of a popup with higlight text

![image](https://github.com/odoo/odoo/assets/52911687/f06ffaaf-c46c-4988-956b-0ea715740dbe)